procrastinate/aiopg_connector.py

```diff
@@ -24,6 +24,10 @@ def get_connection(dsn="", **kwargs) -> Awaitable[aiopg.Connection]:
     return aiopg.connect(dsn=dsn, **kwargs)
 
 
+def connection_is_open(connection: aiopg.Connection) -> bool:
+    return not connection.closed
```

Comment:
Is there a rationale to create a separate function for this? :)

Reply:
See the discussion with @ewjoachim above. The functions above the `PostgresJobStore` class definition are the ones dealing with aiopg objects.

Reply:
Ok :+1: .

Reply:
If you both think it's a bad idea, maybe it's time I reconsider :D

Reply:
I think it does not hurt at this point really.

Reply:
I find it a bit overzealous but it makes sense, so it's a +0 for me :) .